assets/js/googlesitekit/modules/datastore/sharing-settings.test.js

```diff
@@ -821,5 +821,173 @@ describe( 'core/modules sharing-settings', () => {
 				).toBe( true );
 			} );
 		} );
+
+		describe( 'haveModuleSharingSettingsChanged', () => {
+			const moduleSlug = 'search-console';
+			it( 'requires the moduleSlug param', () => {
+				expect( () => {
+					registry
+						.select( CORE_MODULES )
+						.haveModuleSharingSettingsChanged();
+				} ).toThrow( 'moduleSlug is required.' );
+			} );
+
+			it( 'should return undefined if `sharingSettings` or `savedSharingSettings` cannot be loaded', () => {
+				global[ dashboardSharingDataBaseVar ] = undefined;
+
+				expect(
+					registry
+						.select( CORE_MODULES )
+						.haveModuleSharingSettingsChanged( moduleSlug )
+				).toBeUndefined();
+			} );
+
+			it( 'informs whether client-side sharing-settings differ from server-side ones for the given module', () => {
```

Comment:
Remove hyphen for consistency
```suggestion
			it( 'informs whether client-side sharing settings differ from server-side ones for the given module', () => {
```